readthedocs/projects/views/private.py

```diff
@@ -1239,12 +1232,15 @@ def _get_csv_data(self):
 
     def _get_retention_days_limit(self, project):
         """From how many days we need to show data for this project?"""
-        return settings.RTD_ANALYTICS_DEFAULT_RETENTION_DAYS
+        return PlanFeature.objects.get_feature_value(
+            project,
+            type=self.feature_type,
+            default=settings.RTD_ANALYTICS_DEFAULT_RETENTION_DAYS,
```

Comment:
In the same way we have `self.feature_type`, I think we should have `self.feature_type_default` and use them here. That way, the setting, and its default are defined in the same place.

Reply:
Defining that value at the class level could make it hard to test, since the value will be evaluated at import time, changing the setting after that won't have any effect (don't think we have tests overriding that setting currently, but we have been bitten by this in the past).